src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,23 @@
+import "./modal.css";
+
+type ModalProps = {
+  children?: React.ReactChild;
+  isOpen: boolean;
+  closeModal: () => void;
+};
+
+export default function Modal({ children, isOpen, closeModal }: ModalProps) {
+  const modalClickHandler = (e: React.SyntheticEvent) => {
+    if (e.target === e.currentTarget) {
+      closeModal();
+    }
+  };
+  return (
+    <div
+      className={!!isOpen ? "modal open" : "modal"}
```

Comment:
You have used double NOT(!) operator. You can remove this and it will still work the same right?

Reply:
Yes, you are correct, as it is already typed as boolean, this will be redundant